newton/_src/solvers/implicit_mpm/rasterized_collisions.py

```
@@ -140,18 +133,12 @@ def collision_sdf(
-    )
-
-    min_sdf = ground_sdf
-    sdf_grad = collider.ground_normal
+    min_sdf = float(_INFINITY)
+    sdf_grad = wp.vec3(0.0)
     sdf_vel = wp.vec3(0.0)
     closest_point = wp.vec3(0.0)
-    collider_id = int(_GROUND_COLLIDER_ID)
-    material_id = int(_GROUND_COLLIDER_MATERIAL_INDEX)
+    collider_id = int(_NULL_COLLIDER_ID)
+    material_id = int(-1)
```

Ruff (0.14.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)